web/rootfs/defaults/meet.conf

```diff
@@ -130,6 +131,20 @@ location ^~ /etherpad/ {
 }
 {{ end }}
 
+{{ if .Env.WHITEBOARD_URL_BASE }}
+# whiteboard (excalidraw-backend)
+location = /socket.io/ {
```

Comment:
Let's add some extra path here like excalidraw-collab/socket.io sinilar to what we do for Etherpad.